pkgs/tools/security/der-ascii/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule {
+  pname = "der-ascii";
+  version = "unstable-2022-01-18";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "der-ascii";
+    rev = "a31891b88eb633a70d69e413cb4061a2576ebecc";
+    sha256 = "0pm368nim12akp55x13nk8vzjw92yzj27jvqsckgbzws5vbgrvld";
+  };
+  vendorSha256 = "0sjjj9z1dhilhpc8pq4154czrb79z9cm044jvn75kxcjv6v5l2m5";
+
+  meta = with lib; {
+    description = ''
+      A small human-editable language to emit DER or BER encodings of ASN.1
+      structures and malformed variants of them
+    '';
+    homepage = "https://github.com/google/der-ascii";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ alexshpilkin ];
```

Comment:
@hawkw and myself are both interested in helping get this tool packaged having duplicated each other's work to write local derivations for use w/ various flakes. We're both in the maintainers list if you would like help.

Reply:
I’ll just plop both of you into the list then, if you don’t mind :) My use of this tool was honestly quite situational and I don’t expect to be keeping careful track of it, I just had the packaging written up anyway and thought I might as well submit it.

Reply:
... Incidentally, any chance I could interest either one of you in my `dumpasn1` PR <https://github.com/NixOS/nixpkgs/pull/215128>? :)

Reply:
> I’ll just plop both of you into the list then, if you don’t mind :) My use of this tool was honestly quite situational and I don’t expect to be keeping careful track of it

I definitely don't mind and keep a pretty close watch on the upstream. I use this tool a lot and expect to build more things on top of it in the near future.

> ... Incidentally, any chance I could interest either of you in my dumpasn1 PR https://github.com/NixOS/nixpkgs/pull/215128? :)

I'm less keen on this one It looks like it's from another era of software development practices and I don't have any immediate use for it myself. Sorry :sweat_smile: